grid/app/main/dfl/auth/__init__.py

```diff
@@ -0,0 +1,64 @@
+import json
+import functools
+from flask_login import LoginManager, current_user, UserMixin
```

Comment:
You should put a blank line between each group of imports.

Imports should be grouped in the following order:

1.Standard library imports.
2.Related third party imports.
3.Local application/library specific imports.

Reply:
I recommend to include https://github.com/pre-commit/mirrors-isort in pre-commit configuration
This automatically sort the imports.

Reply:
Seems a good idea to me.
Perhaps, We can also research/test [google/yapf](https://github.com/google/yapf) as suggested by @AmrMKayid